cpp/log/test.cpp

```diff
@@ -61,4 +68,76 @@ void test() {
 	t1.join();
 }
 
+/**
+ * Creates a FileSink to also write log Messages
+ * into /tmp/openage-log-test.
+ * Only Messages with a level equals or higher
+ * than level::err will be printed in that file.
+ */
+void file_log_sink () {
+	const char *filelog_name = "/tmp/openage-log-test";
+	TestLogSource logger;
+	FileSink filelog(filelog_name, true);
```

Comment:
Will do this too then (since I delete the file at the end of the test it doesn't lead to wrong results right now).